test_scripts/Defects/4_6/2322_SDL_must_set_media_app_to_LIMITED_and_AUDIBLE_due_to_active_embedded_navigation.lua

```diff
@@ -0,0 +1,89 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/SmartDeviceLink/sdl_core/issues/2322
+--
+-- Precondition:
+-- 1) SDL and HMI are started.
+-- 2) Media app is registered and activated.
+-- Description:
+-- SDL must set media app to LIMITED and NOT_AUDIBLE due to active embedded navigation
+-- Steps to reproduce:
+-- 1) Media app in FULL and AUDIBLE and SDL receives OnEventChanged (EMBEDDED_NAVI, isActive=true) from HMI.
+-- Expected result:
+-- SSDL must send OnHMIStatus (LIMITED, NOT_AUDIBLE) to mobile app.
+-- Actual result:
+-- SDL sends OnHMIStatus (BACKGROUND, NOT_AUDIBLE) to mobile app.
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/Defects/commonDefects')
+local mobile_session = require('mobile_session')
+
+--[[ Local Variables ]]
+config.application1 = {
```

Comment:
There is no need to redefine all params for application1.
It is enough to redefine:
```
config.application1.registerAppInterfaceParams.isMediaApplication = true
config.application1.registerAppInterfaceParams.appHMIType = { "MEDIA" }
```